oak-blob-cloud-azure/src/main/java/org/apache/jackrabbit/oak/blob/cloud/azure/blobstorage/AzureBlobStoreBackend.java

```diff
@@ -130,36 +118,19 @@ public class AzureBlobStoreBackend extends AbstractSharedBackend {
     public void setProperties(final Properties properties) {
         this.properties = properties;
     }
+    private volatile BlobContainerClient azureContainer = null;
 
-    private volatile CloudBlobContainer azureContainer = null;
-
-    protected CloudBlobContainer getAzureContainer() throws DataStoreException {
+    protected BlobContainerClient getAzureContainer() throws DataStoreException {
         if (azureContainer == null) {
             synchronized (this) {
                 if (azureContainer == null) {
-                    azureContainer = azureBlobContainerProvider.getBlobContainer(getBlobRequestOptions());
+                    azureContainer = azureBlobContainerProvider.getBlobContainer(retryOptions, isProxyNeeded, properties);
                 }
             }
         }
         return azureContainer;
     }
 
-    @NotNull
-    protected BlobRequestOptions getBlobRequestOptions() {
-        BlobRequestOptions requestOptions = new BlobRequestOptions();
-        if (null != retryPolicy) {
-            requestOptions.setRetryPolicyFactory(retryPolicy);
-        }
-        if (null != requestTimeout) {
-            requestOptions.setTimeoutIntervalInMs(requestTimeout);
-        }
-        requestOptions.setConcurrentRequestCount(concurrentRequestCount);
-        if (enableSecondaryLocation) {
-            requestOptions.setLocationMode(LocationMode.PRIMARY_THEN_SECONDARY);
```

Comment:
I see you removed the secondary location concept. How was this implemented in azure sdk v12 ? because I cannot see it anyware